features/edit_dish.feature

```diff
@@ -29,13 +29,12 @@ Feature: As a restaurant owner
       | Bosse | bosse@food.se | owner |
     And I log in as "Bosse"
     And I visit the restaurant page for "Anna"
-    And I click the link "Edit"
-    Then I should see "You are not authorized to access this page."
+    Then I should not see "Edit dish"
```

Comment:
So the conditional display of the Edit button is one thing but I would also like to see a test for what is happening when a user tries to access the route for a dish that is not his own directly.